index.js

```diff
@@ -69,8 +69,9 @@ function exit() {
 
 globals.setImmediate(function () {
 	var numberOfTests = runner.select({type: 'test', skipped: false}).length;
+	var numberOfSkippedTests = runner.select({type: 'test', skipped: true}).length;
```

Comment:
So that should be the only `select` you need. @vdemedes - do we need that first value for a reporter?